src/main/kotlin/no/nav/soknad/arkivering/soknadsfillager/repository/FilRepository.kt

```diff
@@ -1,7 +1,13 @@
 package no.nav.soknad.arkivering.soknadsfillager.repository
 
 import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.data.jpa.repository.Query
 import org.springframework.stereotype.Repository
 
 @Repository
-interface FilRepository : JpaRepository<FilDbData, String>
+interface FilRepository : JpaRepository<FilDbData, String> {
+
+	@Query(value = "SELECT count(id) FROM documents where document is not null", nativeQuery=true)
+	fun documentCount(): Long?
```

Comment:
Kan queryn returnere null, eller kunde vi endre returtypen til `Long` istedenfor `Long?` ?

Reply:
Jeg tror det stemmer at den alltid vil returnere en verdi. Testet ved å kjøre lokalt mot tom database, og fikk 0 i retur. Endrer til **Long** fra **Long?**

Reply:
Supert. Då kan du ju fjerne null-sjekken i `FileMetrics.filesInDbGaugeSet()` også, så er det bare å mergea in i main sen :)